Slack.Automation/Promact.Erp.Web/Promact.Erp.Web.csproj

```diff
@@ -996,7 +1000,7 @@
     <Content Include="Views\Home\AfterLogIn.cshtml" />
     <Content Include="tsconfig.json" />
     <Content Include="typings.json" />
-    <None Include="Properties\PublishProfiles\Local.pubxml" />
+	<None Include="Properties\PublishProfiles\Local.pubxml" />
```

Comment:
this was already in the project - [Promact.Erp.Web.csproj - dev branch](https://github.com/Promact/slack-productivity-bots/blob/dev/Slack.Automation/Promact.Erp.Web/Promact.Erp.Web.csproj#L999)